app/resources/[slug]/page.tsx

```
@@ -0,0 +1,147 @@
+            <div className="flex items-center">
+              <Tag className="w-4 h-4 mr-1 text-orange-500" />
+              <span className="text-sm font-medium text-orange-500 bg-orange-100 dark:bg-orange-900 dark:text-orange-300 px-2 py-1 rounded-full">
+                {article.category}
+              </span>
+            </div>
+          </div>
+        </div>
+
+        <div className="bg-white dark:bg-gray-800 rounded-lg shadow-lg p-8">
+          <article className="prose prose-orange max-w-none dark:prose-invert prose-headings:text-gray-900 dark:prose-headings:text-white prose-a:text-orange-500 hover:prose-a:text-orange-600">
+            <div dangerouslySetInnerHTML={{ __html: article.contentHtml }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)